### ISSUE TITLE
JQuery dependency no in packages or system config? 

### ISSUE THREAD

**Issue by ddivita** [sstorie/experiments#23]
I don't see JQuery setup as a dependency in the config or packages,json. Also, I do not see how signalr is supposed to be registered in the systemjs.config.js. Am I missing something?

**Comment by sstorie** [comment]
I assume this is a question about the SignalR work?

**Comment by ddivita** [comment]
Yes, sorry about that. I didn't realize you had other projects under the
same repo.

**Comment by sstorie** [comment]
If memory serves me right, jquery is pulled in as a dependency of the [signalr package](https://github.com/sstorie/experiments/blob/master/angular2-signalr/client/package.json#L30), and referenced directly from node_modules in [index.html](https://github.com/sstorie/experiments/blob/master/angular2-signalr/client/index.html#L17).

Does that help at all?

**Comment by ddivita** [comment]
OK. I guess I was trying to be a purist. I wanted to make sure since in
other projects I have had to specifically configure JQuery to run with
angular 2 / 4

**Comment by sstorie** [comment]
Yeah, sorry I know that example hasn't been updated for quite some time.
